docs/content/error/$compile/badrestrict.ngdoc

```diff
@@ -0,0 +1,12 @@
+@ngdoc error
+@name $compile:badrestrict
+@fullName Invalid Directive Restrict
+@description
+
+This error occurs when the restrict property of a directive is not valid.
+
+The directive restrict property must be a string with possible character values being:
```

Comment:
I would say _"...string including one of more of the following characters:"_.
It might be also worth adding a couple of example: `restrict: 'E'`, `restrict: 'EAC'`